### ISSUE TITLE
Combine connection's `closed_reason` and `is_closed` properties

### ISSUE THREAD

**Issue by mehaase** [HyperionGray/trio-websocket#70]
As I'm writing docs, I realize these separate properties are redundant. I originally added `is_closed` because I didn't like the way that `if ws.close_reason:` reads, but I think renaming `closed_reason` to `closed` (still returns a `CloseReason` object) and removing `is_closed` is better. Example:

```python
if ws.closed:
  print('Connection closed!', ws.closed.code, ws.closed.reason)
```

I like the way that this reads.

**Comment by njsmith** [comment]
Writing docs is a fabulous way to improve APIs, isn't it :-)

**Comment by belm0** [comment]
I'm neutral on separating them-- `closed` sounds like a boolean, and `close_reason` on its own sounds only ancillary.    The best I can think of for a combined attribute is `closed_as`, but it's not much better.

**Comment by mehaase** [comment]
Closed via #81.
